config/default.yml

```diff
@@ -838,6 +838,11 @@ PrePush:
     command: ['ruby', '-Ilib:test', '-rminitest', "-e 'exit! Minitest.run'"]
     include: 'test/**/*_test.rb'
 
+  TestUnit:
+    enabled: false
+    description: 'Run test-unit'
+    command: ['rake', 'test']
```

Comment:
I'm not very familiar with `rake test`, but couldn't it be configured to run anything and not just test-unit? If that is the case, and probably even if it isn't, this hook should be called `RakeTest` instead since that is what it will be running.